kubernetes_cluster_autoscaler/tests/test_e2e.py

```diff
@@ -0,0 +1,22 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from datadog_checks.base.constants import ServiceCheck
+from datadog_checks.dev.utils import get_metadata_metrics
+
+from . import common
+
+
+@pytest.mark.e2e
+def test_check_kubernetes_cluster_autoscaler_e2e(dd_agent_check, instance):
+    aggregator = dd_agent_check(instance, rate=True)
+    metrics = common.METRICS_MOCK
+
+    for metric in metrics:
+        aggregator.assert_metric(name=metric)
+
+    aggregator.assert_all_metrics_covered()
+    aggregator.assert_metrics_using_metadata(get_metadata_metrics())
```

Comment:
I would ditch all these assertions and keep only the service check. The gain of asserting histogram metrics isn't worth the risk of flakiness in my experience.

I would even consider ditching the service check type. This test is really mostly a smoke test: can we load the check into the agent and the agent execute it? For functionality testing, use unit tests. For instance, in the unit tests you can assert that a metric of type `HISTOGRAM` was submitted, that should be enough for us.

Reply:
noted